static/src/javascripts/projects/common/modules/experiments/tests/contributions-embed.js

```diff
@@ -0,0 +1,95 @@
+define([
+    'bean',
+    'qwery',
+    'common/utils/$',
+    'common/utils/template',
+    'common/views/svg',
+    'common/utils/fastdom-promise',
+    'common/utils/mediator',
+    'text!common/views/contributions-embed.html',
+    'common/utils/robust'
+
+], function (bean,
+             qwery,
+             $,
+             template,
+             svg,
+             fastdom,
+             mediator,
+             contributionsEmbed,
+             robust
+) {
+
+
+    return function () {
+
+        this.id = 'ContributionsEmbed20160823';
+        this.start = '2016-08-23';
+        this.expiry = '2016-08-30';
+        this.author = 'Jonathan Rankin';
+        this.description = 'Test contributions embed with amount picker.';
+        this.showForSensitive = false;
+        this.audience = 0.13;
+        this.audienceOffset = 0.05;
+        this.successMeasure = 'Determine the effectivness of allowing user to pick contribution amount in the cta';
```

Comment:
that's not a measure, you need to measure something like pv per visit, or I suppose average fall off rate between clicking the button and paying?